### PR TITLE
Handler error when kubeconfig file not exist on kubeadm reset

### LINKED ISSUE #1217: kubeadm reset is not idempotent
## Is this a BUG REPORT or FEATURE REQUEST?

BUG REPORT

## Versions

1.13.alpha.3, at cfd53d6dae55d8663b082ba5ab2bd6a6dc01b24f

**Environment**:
- **Kubernetes version**: 1.12.2
- **OS**: openSUSE/Leap
- **Kernel**: Linux laxe 4.12.14-lp150.12.22-default #1 SMP Sat Oct 13 05:05:16 UTC 2018 (09415e8) x86_64 x86_64 x86_64 GNU/Linux

## What happened?

I did `kubeadm reset --cri-socket=...` for `crio` when I realized I did not create my cluster with `crio` but  with `docker`. However, a second `kubeadm reset` with the right parameters failed because `kubeconfig` was removed on the first run.

## What you expected to happen?

I expected `kubeadm reset` to be idempotent and to go ahead with any neccessary cleanups, ignoring errors because of things that have already been reset. In my case, I expected to be able to run `kubeadm reset` again and `kubeadm` to stop the docker containers.

## How to reproduce it (as minimally and precisely as possible)?

Invoke `kubeadm reset` twice.

### PR DESCRIPTION
**Pull request by yagonobre**
**What type of PR is this?**
/kind bug
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubernetes/kubeadm/issues/1217

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/sig cluster-lifecycle
/priority important-soon
@kubernetes/sig-cluster-lifecycle-pr-reviews

### CODE REVIEW

**Review by neolit123 — APPROVED** [review]
SGTM thanks for the fix @yagonobre

File: cmd/kubeadm/app/cmd/reset_test.go [hunk 1]
```diff
@@ -273,30 +273,38 @@ func TestGetEtcdDataDir(t *testing.T) {
...
+			validClient:   true,
 		},
 		"etcd pod spec without data volume": {
-			dataDir:       "",
 			podYaml:       etcdPodWithoutDataVolume,
 			expectErr:     true,
 			writeManifest: true,
+			validClient:   true,
+		},
+		"kubeconfig file dont't exist": {
```
Comment (ereslibre) [inline cmd/kubeadm/app/cmd/reset_test.go]: minor, typo: `kubeconfig file doesn't exist`

**Review by fabriziopandini — APPROVED** [review]
@yagonobre thanks!
/approve
after rebase ping me for lgtm

File: cmd/kubeadm/app/cmd/reset_test.go (hunk 1 above)
Comment (yagonobre) [reply cmd/kubeadm/app/cmd/reset_test.go]: Thanks @ereslibre, fixed!

**Pull request merged**